grails-app/controllers/au/org/ala/dataqualityfilter/QualityProfileController.groovy

```diff
@@ -162,7 +162,7 @@ class QualityProfileController extends RestfulController<QualityProfile> {
                     )
             ]
     )
-    @Path("/api/v1/data-profiles/{id}")
+    @Path("/api/v1/data-profiles/{profileId}")
```

Comment:
Using profileId in @path, instead of id, will it affect how the router parses the parameters?

Reply:
This change is purely for openapi spec. `@Path` does not impact `UrlMappings`.